mindsdb/interfaces/model/model_controller.py

```diff
@@ -2,6 +2,7 @@
 import sys
 import json
 import base64
+import datetime as dt
```

Comment:
there is an `import` statement with `datetime` couple lines below. Please modify it instead of creating a duplicate import statement

Reply:
fixed